### PR TITLE
feat(environments): Search bar supports excluding the environment tag

### PR DESCRIPTION
**Pull request by lynnagara**

### CODE REVIEW

File: tests/js/spec/views/stream/searchBar.spec.jsx [hunk 1]
```diff
@@ -1,22 +1,37 @@
 import React from 'react';
 import {shallow, mount} from 'enzyme';
 
-import {Client} from 'app/api';
 import SearchBar from 'app/views/stream/searchBar';
 import TagStore from 'app/stores/tagStore';
 
-describe('SearchBar', function() {
+fdescribe('SearchBar', function() {
```
Comment (billyvg) [inline tests/js/spec/views/stream/searchBar.spec.jsx]: `fdescribe`

File: src/sentry/static/sentry/app/views/stream/searchBar.jsx [hunk 2]
```diff
@@ -13,25 +13,28 @@ import ApiMixin from '../../mixins/apiMixin';
...
-
     defaultQuery: PropTypes.string,
     query: PropTypes.string,
     defaultSearchItems: PropTypes.array.isRequired,
     disabled: PropTypes.bool,
     placeholder: PropTypes.string,
-
-    onQueryChange: PropTypes.func,
     onSearch: PropTypes.func,
+    excludeEnvironment: PropTypes.bool,
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/stream/searchBar.jsx]: Can we add a quick comment on what this prop will do?

Comment (billyvg) [inline src/sentry/static/sentry/app/views/stream/searchBar.jsx]: Can we add a quick comment on what this prop will do?

**Review by billyvg — APPROVED** [review]

File: tests/js/spec/views/stream/searchBar.spec.jsx (hunk 1 above)
Comment (lynnagara) [reply tests/js/spec/views/stream/searchBar.spec.jsx]: oops, thanks

Comment (lynnagara) [reply tests/js/spec/views/stream/searchBar.spec.jsx]: Weird that it passed CI

**Pull request merged**